jstest/testrunner/devices/ssh_device.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from threading import Thread
+
+from jstest.common import console, utils, paths
+from jstest.testrunner.devices.device_base import RemoteDevice
+from jstest.testrunner.devices.connections.sshcom import SSHConnection
+from jstest.testrunner import utils as testrunner_utils
+
+
+class SSHDevice(RemoteDevice):
```

Comment:
A minimal class description could be cool (to be consistent to other classes) :)